runtime/compiler/build/toolcfg/zos-xlc/common.mk

```diff
@@ -35,12 +35,13 @@ DEPSUFF=.depend.mk
 ifeq ($(VERSION_MAJOR),8)
 	ARCHLEVEL=7
 	TGTLEVEL=zOSV1R13
+	TUNELEVEL?=10
 else
 	ARCHLEVEL=10
 	TGTLEVEL=zOSV2R3
+	TUNELEVEL?=12
 endif
```

Comment:
Tabs should not be used for indentation here (to avoid `make` treat them as part of a recipe).

Reply:
Removed the tabs